### ISSUE TITLE
Empty spaces break media queries

### ISSUE THREAD

**Issue by yyss8** [scssphp/scssphp#622]
Hi, we are trying to switch the scss compiler in our project from `node-sass` to `scssphp` but I noticed none of the media rules are being compiled correctly.

We are using the helper function from the  `breakpoint-sass` library in our project to generate all the media rules like this

<img width="292" alt="image" src="https://user-images.githubusercontent.com/13445723/191347082-08a4d8b6-ae12-4e9e-9754-90f4be4e64c0.png">

However, the argument `123px` that we passed to the `breakpoint` function is being ignored in the compiled css for some reason.

<img width="234" alt="image" src="https://user-images.githubusercontent.com/13445723/191347244-5c5aba02-be88-4af8-833d-8e9edc0318f5.png">

After doing some more debugs, I noticed there are 2 leading spaces in the query string generated by `breakpoint-sass` and it looks like `scssphp` won't be able to compile the media rules correctly when there are either leading or trailing spaces in the media queries.

Source:
<img width="352" alt="image" src="https://user-images.githubusercontent.com/13445723/191347928-8a1b92ce-95b5-499c-8153-c26343960999.png">

Result:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/13445723/191348024-454115a7-44e1-4235-956f-7e4784883f21.png">

**Comment by stof** [comment]
I'll try to fix that, but I cannot guarantee I'll be able to fix it in 1.x (due to #157 and our existing implementation of media queries).
Anyway, this will be fixed for sure in 2.0 once it is ready (the big focus of 2.0 is correctness).

Could you provide the reproducing case as a markdown code snippet (allowing to copy it to reproduce it) rather than as a screenshot of your IDE ?

**Comment by yyss8** [comment]
> I'll try to fix that, but I cannot guarantee I'll be able to fix it in 1.x (due to #157 and our existing implementation of media queries). Anyway, this will be fixed for sure in 2.0 once it is ready (the big focus of 2.0 is correctness).
> 
> Could you provide the reproducing case as a markdown code snippet (allowing to copy it to reproduce it) rather than as a screenshot of your IDE ?

Thanks for the quick response

Here's the reproducing case that I was using:
```scss
// Leading space.
@media #{" (max-width: 300px)"} {
  font-size: 10px;
}

// No space.
@media #{"(max-width: 300px)"} {
  font-size: 10px;
}

// Trailing Space.
@media #{"(max-width: 300px) "} {
  font-size: 10px;
}

```

**Comment by stof** [comment]
FYI, it looks like the trailing spaces are now working in the latest version.
